.golangci.yml

```diff
@@ -27,6 +27,8 @@ issues:
       text: "\"io/ioutil\" has been deprecated"
     - linters: [errcheck]
       text: "tx\\.Rollback"
+    - linters: [noctx]
+      path: "_test\\.go"
```

Comment:
Using a context for `http.Request` isn't always necessary in tests, so I think we can ignore issues from this linter in test files.